### PR TITLE
Image component

### PR DESCRIPTION
**Pull request by prateek1361**

### CODE REVIEW

**Review by Rohitprasad83 — COMMENTED** [review]
Add a preview link or screenshot of the working component.

File: components/badge.html [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+        
+      </div>
+      <div class="main">
+      <h3>Badge on Avatars</h3>
+      
+      <div class="component-area">
+        
+          
+          <div class="avatar-w-badge">
+            <img class="round-avatar-badge" src="https://images.unsplash.com/photo-1438761681033-6461ffad8d80?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=870&q=80" alt="">
```
Comment (Naveenchandar) [inline components/badge.html:76]: Please provide "alt" description for accessibility

File: components/badge.html [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+            <span class="badge badge-online"></span>
+        </div>
+
+        <div class="avatar-w-badge">
+            
+            <img class="round-avatar-badge"  src="https://images.unsplash.com/photo-1438761681033-6461ffad8d80?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=870&q=80" alt="">
```
Comment (Naveenchandar) [inline components/badge.html:82]: Please provide "alt" description for accessibility

File: components/badge.html [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+            <span class="badge badge-offline"></span>
+        </div>
+         </div>
+         
+         <h4>Badge on Icons</h4>
+         <div class="component-area">
+            <div class="icon-w-badge">
+             
+                <img class="icon-badge" src="https://media.istockphoto.com/vectors/shopping-cart-icon-isolated-on-white-background-vector-id1206806317?s=612x612"alt="">
```
Comment (Naveenchandar) [inline components/badge.html:91]: Same as above

File: components/badge.html [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+         </div>
+         <div><img class='code-img' src="../assets/badge.png"/></div>
+    
+
+
+      
+            
+                
+            
+
```
Comment (Naveenchandar) [inline components/badge.html:110]: Pleas remove the empty space

File: main.css [hunk 5]
```diff
@@ -0,0 +1,277 @@
+:root{
+    --primary:skyblue;
+    --error:red;
+    --warning:yellow;
+    --success:green;
+    --secondary:white;
+    --online-badge:green;
+    --offline-badge:grey;
+    font-size: 10px;
```
Comment (Naveenchandar) [inline main.css:9]: Why px instead of relative units ? Any reason ?

File: main.css [hunk 6]
```diff
@@ -0,0 +1,277 @@
...
+    border-radius: 50%;
+    position: absolute;
+    top: -3px;
+    right: -1px;
+    background-color: rgb(0, 195, 255);
+  }
+  #buttons{
+    margin:0 auto;    
+    max-width:60rem;
+    display: flex;
```
Comment (Naveenchandar) [inline main.css:153]: Suggestion:
If flex is used often, you can declare a variable or separate class name.

File: main.css [hunk 7]
```diff
@@ -0,0 +1,277 @@
...
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        gap: 2rem;
+      }
+      
+   .avatar-w-badge{
+    position: relative;
+    
+  }
```
Comment (Naveenchandar) [inline main.css:106]: Please format the code.

File: main.css [hunk 8]
```diff
@@ -0,0 +1,277 @@
...
+    background-color: red;
+}
+.safe-solid{
+    background-color: green;
+    
+}
+
+
+.primary-outline{
+    border:2px solid#49a0eb ;
```
Comment (Naveenchandar) [inline main.css:204]: space is missing between border properties.Please once verify it.

File: main.css [hunk 9]
```diff
@@ -0,0 +1,277 @@
...
+
+
+
+
+
+
+
+  
+
+
```
Comment (Naveenchandar) [inline main.css:277]: Please remove empty spaces

**Pull request merged**